netebpfext/net_ebpf_ext_drv.c

```diff
@@ -113,61 +116,43 @@ __net_ebpf_ext_driver_initialize_objects(
         goto Exit;
     }
 
-    device_create_flag = TRUE;
+    _net_ebpf_ext_driver_device_object = WdfDeviceWdmGetDeviceObject(_net_ebpf_ext_device);
 
-    status = net_ebpf_ext_register_providers();
-    if (!NT_SUCCESS(status)) {
-        goto Exit;
-    }
+    net_ebpf_ext_register_callouts(_net_ebpf_ext_driver_device_object);
+    // ignore status. at boot, registration can fail.
+    // we will try to re-register during program load.
```

Comment:
```suggestion
    (void)net_ebpf_ext_register_callouts(_net_ebpf_ext_driver_device_object);
    // Ignore status. At boot, registration can fail.
    // We will try to re-register during program load.
```

Reply:
This comment is wrong. I have filed an issue.